src/js/app.js

```diff
@@ -0,0 +1,69 @@
+$(document).ready(function() {
+// start without exit button
+  $('#exit').hide();
+  // Variables
+  var myInterval;
+  var intervalID = 0;
+  // Functions
+  function runShow(pictures) {
+    // $('#pictureResults').html('<img src="' + pictures[intervalID] + '">');
+    $('#pictureResults').attr('src', pictures[intervalID]);
+    console.log(intervalID);
+  }
+  function startShow(pictures) {
```

Comment:
When starting the show, you'll also want to show the image element. Your reset button is hiding the image.